axelrod/strategies/axelrod_first.py

```diff
@@ -487,3 +489,69 @@ class UnnamedStrategy(Player):
     def strategy(opponent: Player) -> Action:
         r = random.uniform(3, 7) / 10
         return random_choice(r)
+
+@FinalTransformer((D, D), name_prefix=None)
+class SteinAndRapoport(Player):
+    """
```

Comment:
This isn't quite formatted correctly (see: http://axelrod.readthedocs.io/en/stable/tutorials/contributing/strategy/docstrings.html).

Can you remove one indentation level for the description (it should be at the same level as the `"""`).

Then can you add:

```
Names:
- SteinAndRapoport [Axelrod1980]_
```